### PR TITLE
Calculate the average

### LINKED ISSUE #1: Add an average button
As long as the cards aren't too far off, we take the average and use that as an estimate. 

It would be great if the app calculated the average automatically. Will send a pull request when I get a chance. For now, here's a snippet of code I just threw together than can be pasted in the console.

var cards = $('.cards > div > div'); alert('Average is ' + Math.round(cards.reduce(function (a, b) { return {innerText: (+a.innerText||0) + (+b.innerText||0)}; }).innerText / cards.length))

PS - Thank you for making this app. Very well done and useful!

### PR DESCRIPTION
**Pull request by pmcelhaney**
Fixes #1. Shows the average after you're finished voting.

### CODE REVIEW

File: app/js/controllers.js [hunk 1]
```diff
@@ -98,6 +102,9 @@ function RoomCtrl($scope, $routeParams, $timeout, socket) {
     $scope.placeholderVotes = voteArr;
 
 
+    var total =  _.reduce(_.map(_.pluck($scope.votes, 'vote'), parseFloat), sumOfTwo, 0);
```
Comment (jh185173) [inline app/js/controllers.js:105]: There's a bug here.  parseFloat will choke if anyone has voted ½ or with a non-numeric vote.  Non-numeric votes should be excluded and ½ translated to 0.5.

**Comment by barthez** [comment]
Average and StdDev calculations are already in master, I believe this PR can be closed.

**Comment by jh185173** [comment]
> Average and StdDev calculations are already in master, I believe this PR can be closed.

Thanks @barhez - Confirming - commit 3de12cca5 looks like it fixed the calculation bug, and this PR looks obsolete.

File: app/js/controllers.js (hunk 1 above)
Comment (pmcelhaney) [reply app/js/controllers.js:105]: This PR was open almost 7 years before it go a review (and another 1.5 years before I noticed). I'm not mad -- that's amazing!

Comment (jh185173) [reply app/js/controllers.js:105]: Amazing but accidental.  From my side, it was meant as a bug report through a non-standard channel than a code review.  I must have encountered the bug in the wild before I even started looking for the root cause.

Comment (pmcelhaney) [reply app/js/controllers.js:105]: Sorry, that came across like a complaint and a I didn't mean for it to be. I just thought it was amusing -- pretty cool actually -- that you saw the PR _and responded to it_ after such a long time. And that it took me such a long time to see that you had responded. 

Open source is hard.

Comment (jh185173) [reply app/js/controllers.js:105]: Hard, and honestly amazing (often amusing, too).  No irony read or intended.

**Pull request closed without merging**